### DISCUSSION TITLE
Stay on the current panel when select async item

### DISCUSSION THREAD

**Discussion by vaynevayne** [JedWatson/react-select#5585] [Q&A]
Now I have to search once, I can only pick one value, and then I have to search again,
Expectation: Search once, can select more than one value
Action: After selecting the value, do not go back to the original panel

https://codesandbox.io/s/codesandboxer-example-forked-hge7rd?file=/example.tsx

**Comment by manjushsh** [comment]
Hi @vaynevayne, Check this modified code. I hope this works for you.
[Code Sandbox link to working example](https://codesandbox.io/s/codesandboxer-example-forked-huhmp2?file=/example.tsx)

**Reply by vaynevayne** [reply]
no, 

<img width="1150" alt="image" src="https://user-images.githubusercontent.com/124666577/225823670-2d64471f-5abb-4f87-982d-eabbee244450.png">

<img width="1195" alt="image" src="https://user-images.githubusercontent.com/124666577/225823653-dafdaa12-a3ce-48c7-9f37-f81ea810ebe6.png">


when i select ocean, i want menu list is `orange` and `slate`, not other

Honestly, this should be the default behavior of a select component, and I don't need to ask

**Reply by manjushsh** [reply]
thats becuse of way filter is written. Let me check it again
